### DISCUSSION TITLE
Unable to download to current Brave browser

### DISCUSSION THREAD

**Discussion by UtterlyDistraught** [Cookie-AutoDelete/Cookie-AutoDelete#1295] [Q&A / Support]
Have had Cookie-AutoDelete in past.  Trying to download to my Brave browser and get message "Failed-Forbidden"  Help!

**Comment by kennethtran93** [comment]
Where are you downloading/installing from?  This link from the Chrome webstore (https://chrome.google.com/webstore/detail/cookie-autodelete/fhcgjolkccmbidfldomjliifgaodjagh) for the latest version should be sufficient, unless brave as some sort of permissions that you need to grant to install this on top of the permissions that this addon requires.

**Comment by UtterlyDistraught** [comment]
I am downloading it from the official site, including using your link.  I still get the message, "Failed.  Forbidden".  I tried it disabling Ghostery and Privacy Badger
but same thing happens.  Browser is Brave.  This is the first time I have had any problem with downloading an extension so I am really at a loss.  It is still
active on older rarely used "Iron" and "Dissenter"  and "Mozilla Firefox browsers so it's not disallowed anywhere else.
Thanks for responding.  Any other suggestions?

________________________________

**Comment by kennethtran93** [comment]
According to a search I did, I came upon this issue brave/brave-browser#16611 which says to update your brave version to at least 1.32 according to that issue a few months ago. Try asking on through that repository as well if need be.

**Comment by UtterlyDistraught** [comment]
Hi and thanks again.  When looking into updating Brave browser, which definitely needs to be done,
 it says to copy all files before removing current Brave browser and reinstalling newest Brave.  I'm a bit embarrassed to admit that I don't
know which files are being referred to/are relevant nor how to save them.  Does it mean all extensions and/or what else?

________________________________

**Comment by kennethtran93** [comment]
Unfortunately you'll need to consult their documentation for this, or get help from other brave users either by searching up how others did it for though their repository for your OS/system.

You'll probably need to at least get a list of your installed and enabled extensions, and within each extension ensure you have a copy/backup of the configuration.  Browser preferences/settings, bookmarks, passwords, etc. should also be backed up just in case.
